RoutesApp_iosTests/AlgorithmTests.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  AlgorithmTests.swift
+//  RoutesApp_iosTests
+//
+//  Created by user on 30/8/22.
+//
+
+import XCTest
+@testable import RoutesApp_ios
+
+class AlgorithmTests: XCTestCase {
+    var line1: Line?
+    var line2: Line?
+
+    override func setUpWithError() throws {
+        // MARK: Route 1
+        let routePoints1 = TestResources.points1Array
+        let stops1 = TestResources.stops1Array
+        line1 = Line(name: "01", categoryRef: "LineCategories/DW8blCpvs0OXwkozaEhn",
+                     routePoints: routePoints1, start: routePoints1[0], stops: stops1)
+        // MARK: Route 2
+        let routePoints2 = TestResources.points2Array
+        let stops2 = TestResources.stops2Array
+        line2 = Line(name: "1001", categoryRef: "LineCategories/JY8blWsvs0OXwkozaJlb",
+                     routePoints: routePoints2, start: routePoints2[0], stops: stops2)
+    }
+
+    override func tearDownWithError() throws {
```

Comment:
No need to add this if you´re not going to use it

Reply:
Removed unnecessary test